plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/manager/GetClientDefaultsCommand.java

```diff
@@ -17,38 +17,28 @@
 // under the License.
 //
 
-package com.cloud.agent.api.guru;
+package com.cloud.agent.api.manager;
 
 import com.cloud.agent.api.Command;
 
-public class ReleaseVmVspCommand extends Command {
+public class GetClientDefaultsCommand extends Command {
 
-    String _networkUuid;
-    String _vmUuid;
-    String _vmInstanceName;
-
-    public ReleaseVmVspCommand(String networkUuid, String vmUuid, String vmInstanceName) {
+    public GetClientDefaultsCommand() {
         super();
-        this._networkUuid = networkUuid;
-        this._vmUuid = vmUuid;
-        this._vmInstanceName = vmInstanceName;
-    }
-
-    public String getNetworkUuid() {
-        return _networkUuid;
     }
 
-    public String getVmUuid() {
-        return _vmUuid;
+    @Override
+    public boolean executeInSequence() {
+        return false;
     }
 
-    public String getVmInstanceName() {
-        return _vmInstanceName;
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
     }
 
     @Override
-    public boolean executeInSequence() {
-        return false;
+    public int hashCode() {
+        return super.hashCode();
     }
-
-}
+}
```

Comment:
No need to override ``equals`` or ``hashCode``.  Simply add a comment that the super class implementations are acceptable because this class does not track any state in addition to the super class.